subgraphs/saucerswap/v2/src/utils/pricing.ts

```diff
@@ -4,42 +4,28 @@ import { Bundle, Pool, Token } from './../types/schema'
 import { BigDecimal, BigInt } from '@graphprotocol/graph-ts'
 import { exponentToBigDecimal, safeDiv } from '../utils/index'
 
-const WETH_ADDRESS = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
-const USDC_WETH_03_POOL = '0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8'
+const WHBAR_ADDRESS = '0x0000000000000000000000000000000000163b5a'
```

Comment:
I'm guessing this is mainnet?
Do we need to support testnet or how might we configure this?

Reply:
Yes is mainnet, will work on `.env` file that provides this values based on network chosen

Reply:
I've done the needed changes so based on the deployed network: `mainnet` or `testnet` it takes some or other values, all is done by `.env` on the subgraph yaml creation using a template, and on `constants.ts` file based on the network the constants are populated with their respective networks, effectively making this subgraph multi-network.

✅ Completed!